apps/fipsinstall.c

```diff
@@ -32,18 +32,26 @@ typedef enum OPTION_choice {
 } OPTION_CHOICE;
 
 const OPTIONS fipsinstall_options[] = {
+    OPT_SECTION("General"),
     {"help", OPT_HELP, '-', "Display this summary"},
+#if 0
+    /* Gross ugly long line that belongs in the manpage. */
```

Comment:
Just delete it and add it to the manpage unless the example is already there.

Reply:
the manpage has a couple of complete examples so i removed this.